module-2/backend/week_two.md

```diff
@@ -5,17 +5,59 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. At a high level, what is ActiveRecord? What does it do/allow you to do?
+
+Active record is a framework for SQL that has methods and allows data to be manipulated more easily.
+
 2. What kind of methods are `belongs_to`, and `has_many`? (i.e. class or instance) Give an example.
+
+pointers? I don't know/ forget the name -  but they point to other objects that they either belong to or have alot of.
+
 3. What do they allow you to do?
+
+They allow you to access the items from another table with the corresponding id.
+
 4. What's the difference between agile workflow and waterfall method?
+
+Agile is a closer production cycle that builds on itself. Waterfall is taking the long shot.
+
 5. What is the difference between `#new` and `#create`?
+
+new is the get method. create is  a post
```

Comment:
I think you're thinking of the methods that we use in our controllers. We were thinking of the ActiveRecord methods. When you call `#new` on an ActiveRecord model you create a new instance of that model, but it does not save to the database until you call `.save`. On the other hand, `#create` creates a new instance and saves it to the DB.